### PR TITLE
Adds a Bicycle to the game

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
add:Adds Bicycles to cargo for 1 million yen
/:cl:

dreamseeker_2017-02-24_18-42-42

### CODE REVIEW

**Comment by TheColdTurtle** [comment]
Pics? And add English please I don't speak moon

**Comment by WJohn** [comment]
goofball is slowly losing his mind

good. join the club.

**Comment by Supermichael777** [comment]
สังวาสคุณผิด

**Comment by Iamgoofball** [comment]
@WJohn these are nienhaus sprites

File: code/modules/vehicles/bicycle.dm [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/obj/vehicle/bicycle
+	name = "bicycle"
+	desc = "Keep away from electricity."
+	icon_state = "bicycle"
+
+var/list/bike_music = list('sound/misc/bike1.mid',
```
Comment (swindly) [inline code/modules/vehicles/bicycle.dm]: 版権の歌はだめだ

Comment (Iamgoofball) [inline code/modules/vehicles/bicycle.dm]: these are high quality originals

**Comment by Repukan** [comment]
Cargo power creep

**Comment by kilog** [comment]
but why

**Comment by Iamgoofball** [comment]
@kilog 
how am i going to get past cycling road without them?

File: code/modules/vehicles/bicycle.dm [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/obj/vehicle/bicycle
+	name = "bicycle"
+	desc = "Keep away from electricity."
+	icon_state = "bicycle"
+	var/easter_egg_chance = 1
+
+var/list/bike_music = list('sound/misc/bike1.mid',
```
Comment (PKPenguin321) [inline code/modules/vehicles/bicycle.dm]: bad indent?

File: code/modules/vehicles/bicycle.dm [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	. = ..()
+
+/obj/vehicle/bicycle/unbuckle_mob(mob/living/buckled_mob,force = 0)
+	if(buckled_mob)
+		buckled_mob << sound(null, repeat = 0, wait = 0, volume = 80, channel = 42)
+	. =..()
+
+/obj/vehicle/bicycle/tesla_act() // :::^^^)))
+	name = "fried bicycle"
+	desc = "Well spent."
```
Comment (PKPenguin321) [inline code/modules/vehicles/bicycle.dm]: what

**Comment by PKPenguin321** [comment]
TOO expensive

File: code/modules/vehicles/bicycle.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/modules/vehicles/bicycle.dm]: nope, global list

File: code/modules/vehicles/bicycle.dm (hunk 3 above)
Comment (Iamgoofball) [inline code/modules/vehicles/bicycle.dm]: the price

File: code/modules/vehicles/bicycle.dm (hunk 2 above)
Comment (RemieRichards) [inline code/modules/vehicles/bicycle.dm]: @Iamgoofball ```/static``` it on the type please.

**Comment by RandomMarine** [comment]
Add a bike voucher that is exchanged for a bike when exported.

**Comment by jughu** [comment]
Only if you cant use it indoors

**Comment by RandomMarine** [comment]
But real talk, I'm not sure if this reference adds anything to the game, especially with only 1% chance of playing the music.

File: code/modules/vehicles/bicycle.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/modules/vehicles/bicycle.dm]: k

**Comment by Quardbreak** [comment]
Please add clown version of this

**Comment by Nienhaus** [comment]
I want to point out, yes they're my sprites but they're also from 2014.

**Comment by ma44** [comment]
@Iamgoofball Your gonna have to bring back seed exports if you expect anyone to buy this

**Comment by Repukan** [comment]
Just use tech disks

**Comment by PKPenguin321** [comment]
i had a dream where somebody else made this PR with no changes and it was merged instantly and it made goof rage

**Comment by Kromgar** [comment]
@Repukan 

You can only sell tech disks once or so i heard

**Comment by bgobandit** [comment]
we already have a bicycle it's the medical doctor

**Comment by Iamgoofball** [comment]
oh my GOD

**Comment by FantasticFwoosh** [comment]
@Repukan once per use of a disk, people abuse especially the multi-tech storage disks all the time, filling the box up with 30 of the blighters for billions of points.

We have enough vehicles now in the code to open a mechanic chop-shop.

File: code/modules/vehicles/bicycle.dm [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/obj/vehicle/bicycle
+	name = "bicycle"
+	desc = "Keep away from electricity."
+	icon_state = "bicycle"
+	var/easter_egg_chance = 1
+
+var/static/list/bike_music = list('sound/misc/bike1.mid',
```
Comment (ChangelingRain) [inline code/modules/vehicles/bicycle.dm]: static ON the type. ON the type.

**Pull request merged**